File: Python Course/Software Development Module - in Python/Revision/Players/FullForwardPlayer.py
"""
FullForwardPlayer.py
Billy Cussen
26/10/2020
"""
import FootballPlayer

class myFullForwardPlayer(FootballPlayer.myFootballPlayer):
    def __init__(self, name):
        super().__init__(name)

    def getRole(self):
        return "Hi "+str(self.name)+", I'm a Full Forward Player"

File: Java/Software Development Module - Java/Revision/Players/FullForwardPlayer.java
/*
FullForwardPlayer.java
Billy Cussen
28/07/2020
*/

public class FullForwardPlayer extends FootballPlayer{

    //Constructor
    public FullForwardPlayer(String name){
        super(name);
    }

    @Override
    public String role(){
        return "Hi, I'm "+name+". I play as a Full Forward.";
    }

}